src/libcore/fmt/mod.rs

```diff
@@ -180,8 +180,9 @@ enum Void {}
            issue = "0")]
 #[doc(hidden)]
 pub struct ArgumentV1<'a> {
-    value: &'a Void,
-    formatter: fn(&Void, &mut Formatter) -> Result,
+    _ph: PhantomData<&'a ()>,
```

Comment:
I don't remember what broke exactly but I just tested it without those changes to `core::fmt` and this time it worked :/